### ISSUE TITLE
How do I make sankey chart to vertically aligned center or middle in div area?

### ISSUE THREAD

**Issue by luceat-lux-vestra** [amcharts/amcharts4#4130]
**Question**
I want to make sankey chart to vertically aligned center/middle in `div` area.
I tried to set some properties but no luck.

// chart.contentValign = "middle";
// chart.verticalCenter = "middle";
// chart.valign = "middle";
// chart.nodeAlign = "middle";
// chart.chartContainer.contentValign = "middle";
// chart.chartContainer.valign = "middle";
// chart.chartContainer.verticalCenter = "middle";

**Environment (if applicable)**
amCharts V4
Chrome 104.0.5112.102

**Additional context**

![test](https://user-images.githubusercontent.com/7198776/185839358-2b5740c2-3360-40f1-a767-231cb4fac7ad.png)

**Comment by martynasma** [comment]
Would you be able to post your whole chart and data on CodePen/jsFiddle so that we can test?

**Comment by luceat-lux-vestra** [comment]
What `Properties` intended to change vertical align of chart?

I tried to make `MWE` on CodePen/jsFiddle but I can't make it.
I'm changing `chart.nodePadding` by row count, gradually.

like `chart.nodePadding  = xx - (x * rowCnt);` and it seems affecting vertical position of chart.
then I added `chart.paddingTop = (xx - (x * rowCnt)) / 2;`
(`xx` and `x` is a number like `44`, `3`)

It makes look better but not perfect.
